runtime/src/server/middleware/get_page_handler.ts

```diff
@@ -356,6 +358,18 @@ export function get_page_handler(
 	};
 }
 
+function replace(
+    template: string,
+    replacers: Record<string, string | CallableFunction>
+) {
+    for (const key in replacers) {
+        if (replacers.hasOwnProperty(key)) {
+            template = template.replace(new RegExp(`%sapper.${key}%`, 'g'), replacers[key] as any);
```

Comment:
Have replacers less coupled to the `%sapper.key%` pattern? It would be flexible to let replacers define their own patterns to replace.

Reply:
I didn't realize that this PR was around when I started my work, but I've done something similar over at #1642, which happens to address your concern here, @arve0